src/explore-education-statistics-frontend/src/components/Feedback.tsx

```diff
@@ -0,0 +1,189 @@
+                <h2 className={styles.heading}>Is this page useful?</h2>
+                <Button
+                  className={styles.button}
+                  variant="secondary"
+                  onClick={handleUsefulFeedback}
+                >
+                  Yes <VisuallyHidden>this page is useful</VisuallyHidden>
+                </Button>
+
+                <Button
+                  className={styles.button}
+                  variant="secondary"
+                  onClick={showNotUseful}
+                >
+                  No <VisuallyHidden>this page is not useful</VisuallyHidden>
+                </Button>
+              </div>
+
+              <div className={styles.feedbackPromptQuestions}>
+                <Button
+                  className={styles.buttonReport}
+                  variant="secondary"
+                  onClick={showProblemEncountered}
+                >
+                  Report a problem with this page
+                </Button>
+              </div>
+            </>
+          )}
+        </div>
+      )}
+
+      {showFormSection && (
+        <div className="govuk-!-padding-left-6 govuk-!-padding-right-6">
+          <h3 className="govuk-!-margin-top-5">
+            Help us improve Explore education statistics
+          </h3>
+          <p>
+            Don't include personal or financial information like your National
+            Insurance number or credit card details.
+          </p>
+
+          <FormProvider>
+            {({ formState }) => {
+              const handleSubmit = async (values: FeedbackRequest) => {
+                await feedbackService.sendFeedback({
+                  url,
+                  userAgent,
+                  response,
+                  context: values.context,
+                  issue: values.issue,
+                  intent: values.intent,
+                });
+
+                showThankYou();
+              };
+
+              return (
+                <Form id="feedbackForm" onSubmit={handleSubmit}>
+                  <FormFieldTextArea<FeedbackRequest>
+                    label="What were you doing?"
+                    name="context"
+                    rows={3}
+                  />
+
+                  {showNotUsefulForm && (
+                    <FormFieldTextArea<FeedbackRequest>
+                      label="What were you hoping to achieve?"
+                      name="intent"
+                      rows={3}
+                    />
+                  )}
+
+                  {showProblemEncounteredForm && (
+                    <FormFieldTextArea<FeedbackRequest>
+                      label="What went wrong?"
+                      name="issue"
+                      rows={3}
+                    />
+                  )}
+
+                  <input type="hidden" name="userAgent" value={userAgent} />
+                  <input type="hidden" name="url" value={url} />
+                  <input type="hidden" name="response" value={response} />
```

Comment:
Would remove these as it's simpler to add these to the directly to the request on form submit.